### ISSUE TITLE
Cannot install PWA studio dependencies from clone

### ISSUE THREAD

**Issue by LordZardeck** [magento-research/pwa-studio#326]
It is not possible to install the dependencies of this repository when cloning.

## This issue is for the following packages:

- [x] `venia-concept`
- [x] `pwa-buildpack`
- [x] `peregrine`
- [x] `pwa-module`
- [x] `pwa-devdocs`

## This issue is a:

- [x] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### Environment

| Question                    |    Answer     |
| --------------------------- | -----------: |
| Magento version             | 2.3.0-beta8 |
| Operating System + version  | macOS Mojave 10.14 |
| node.js version (`node -v`) | v10.6.0 |
| npm version (`npm -v`)      | v6.4.1 |

## Description

Attempting to clone the PWA studio and perform the initial `npm install` command per the instructions on DevDocs, npm errors out with the following error:

```
npm ERR! path *redacted*/pwa-studio/node_modules/.staging/@magento/upward-js-0038374d/node_modules/arr-flatten
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall rename
npm ERR! enoent ENOENT: no such file or directory, rename '*redacted*/pwa-studio/node_modules/.staging/@magento/upward-js-0038374d/node_modules/arr-flatten' -> '*redacted*/pwa-studio/node_modules/.staging/arr-flatten-eb585e55'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!    *redacted*/.npm/_logs/2018-10-04T20_27_58_087Z-debug.log
```

## Expected result:

Npm should install perfectly the first time the repo is cloned without modifications. 

## Possible solutions:

After digging into the issue, i discovered the problem is with the `package-lock.json` configuration. For the `upward-js` package, all it's dependencies are set to be bundled, which they are not. The easiest solution in order to get past this was to delete the `package-lock.json` file in the root of the repository. The proper solution would to be either to actually bundle the dependencies as is expected by `package-lock.json`, or reconfigure the packages to not have their dependencies bundled. 

![2018-10-04_15-40-21](https://user-images.githubusercontent.com/1145989/46501726-e8191500-c7eb-11e8-9575-744c5d2e10bf.png)

**Comment by zetlen** [comment]
I think this may be due to a naive merge made by `npm-merge-driver`. This has happened sevaral times to us locally. Thank you for narrowing it down in that gigantic lockfile!

**Comment by zetlen** [comment]
Fixed in d1ce4aa
